Dockerfile.fuzz

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14
+
+RUN apt-get update && apt-get install -y clang
+COPY . $GOPATH/src/github.com/opencontainers/umoci
+
+RUN cd $GOPATH/src/github.com/opencontainers/umoci && \
+    go clean --modcache && \
+    go mod tidy && \
+    go mod vendor && \
+    rm -r ./vendor
+
+RUN go get -u github.com/dvyukov/go-fuzz/go-fuzz \
+	github.com/dvyukov/go-fuzz/go-fuzz-dep \
+	github.com/dvyukov/go-fuzz/go-fuzz-build
+
+WORKDIR $GOPATH/src/github.com/opencontainers/umoci
```

Comment:
If you put this before the `RUN go clean ...` you don't need to do `cd` in that command.

Reply:
I think this needs to go above line 11 in order to work properly?

Reply:
It should be there now